packages/tiny-mem/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "tiny-mem",
+  "version": "1.0.0",
+  "description": "Minimal, no-dependencies memoization helper",
+  "keywords": [
+    "cache",
+    "function",
+    "memoize"
+  ],
+  "bugs": "https://github.com/bloq/js-packages/issues",
+  "license": "MIT",
+  "author": "Gabriel Montes <gabriel@bloq.com>",
+  "files": [
+    "src"
+  ],
+  "main": "src/index.js",
+  "repository": "bloq/js-packages",
+  "scripts": {
+    "coverage": "nyc npm run test",
+    "test": "mocha"
+  },
+  "nyc": {
+    "all": true,
+    "include": [
+      "src/**/*.js"
+    ],
+    "reporter": [
+      "lcov",
+      "text"
+    ]
+  },
+  "engines": {
+    "node": ">= 14"
+  }
```

Comment:
14 is not even on maintenance mode... Should we use 16 right from scratch? This also applies for other packages..

Reply:
The code is compatible with Node 14 and up. If someone is using that old version, so be it. We should not prevent them from using this package.